test/index.js

```diff
@@ -388,6 +388,21 @@ describe('Bigtable', function() {
       bigtable.createInstance(INSTANCE_NAME, options, assert.ifError);
     });
 
+    it('should respect the type option', function(done) {
+      var options = {type: 'development'};
+      sinon.spy(FakeInstance, 'getTypeType_');
```

Comment:
We need to simulate what this method returns, so we can verify that's what ends up being sent to `request`.

For example, this pattern:

```js
var options = {type: 'development'}
var fakeTypeType = 99

FakeInstance.getTypeType_ = function(options) {
  assert.strictEqual(type, options.type)
  return fakeTypeType
}

bigtable.request = function(config) {
  assert.strictEqual(config.reqOpts.instance.type, fakeTypeType)
}

bigtable.createInstance(INSTANCE_NAME, options, assert.ifError)
```

Reply:
Done